client/src/pages/profile/profile.js

```diff
@@ -30,39 +35,31 @@ const Profile = (props) => {
   };
 
   useEffect(() => {
-    props.fetchUser(userId);
-    props.fetchFollowers(userId);
-    props.fetchFollowing(userId);
+    dispatch(fetchUser(userId));
```

Comment:
This kind of multiple calls can be done in one by having an endpoint something like, "fetchUserInformation" which includes the following and followers information already, this is ok for now, but keep this in mind for the next features